src/server/router/game/history.ts

```diff
@@ -27,13 +30,59 @@ export const historyRouter = createProtectedRouter()
           },
           loser: {
             select: {
+              id: true,
               name: true,
               email: true,
               image: true,
               rating: true,
             },
           },
         },
+      })) as any;
+
+      const winnerPreviousResult = await ctx.prisma.gameResult.findFirst({
```

Comment:
These 3 database queries can be done in parallel in a single prisma transaction to reduce round trips